### PR TITLE
Expose FxOS Add-ons uuid in mini-manifest and API (bug 1216974)

### PR DESCRIPTION
**Pull request by diox**
https://bugzilla.mozilla.org/show_bug.cgi?id=1216974

### CODE REVIEW

File: mkt/extensions/models.py [hunk 1]
```diff
@@ -267,6 +267,7 @@ def mini_manifest(self):
         except ExtensionVersion.DoesNotExist:
             return {}
         mini_manifest = {
+            'id': self.uuid,
```
Comment (diox) [inline mkt/extensions/models.py]: I chose to expose it as `id` to be coherent with signing (which *needs* it to be called `id`). I'm open to comments about why that's a terrible idea.

Comment (chuckharmston) [inline mkt/extensions/models.py]: It's terrible, but fine.

Comment (chuckharmston) [inline mkt/extensions/models.py]: We should mention in documentation endpoint that it's a UUID and not auto-incremented, just for posterity or whatever.

File: mkt/extensions/models.py [hunk 2]
```diff
@@ -613,6 +610,16 @@ def reviewer_sign_if_necessary(self):
         if not private_storage.exists(self.reviewer_signed_file_path):
             self.reviewer_sign_file()
 
+    @property
+    def reviewer_identifier(self):
```
Comment (chuckharmston) [inline mkt/extensions/models.py]: `reviewer_id`? Docstring is great, name is not.

**Comment by chuckharmston** [comment]
r+wc

File: mkt/extensions/models.py (hunk 1 above)
Comment (diox) [inline mkt/extensions/models.py]: I don't think the mini-manifest for add-ons is documented anywhere, except on MDN where it basically says "this is a webapp manifest" :( It's really an internal thing.

**Pull request merged**